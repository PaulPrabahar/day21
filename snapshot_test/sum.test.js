import sum from "./sum";

test("snapshot testing-sum", () => {
    expect(sum(4,5)).toMatchSnapshot();
    expect(sum(3,0)).toMatchSnapshot();
    expect(sum(-1,9)).toMatchSnapshot();
    expect(sum(4,5,5)).toMatchSnapshot();
    expect(sum(4,5,45)).toMatchSnapshot();
})
