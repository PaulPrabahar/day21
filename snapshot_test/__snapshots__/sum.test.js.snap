// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot teasting-sum 1`] = `9`;

exports[`snapshot teasting-sum 2`] = `9`;

exports[`snapshot teasting-sum 3`] = `9`;

exports[`snapshot teasting-sum 4`] = `9`;

exports[`snapshot teasting-sum 5`] = `NaN`;

exports[`snapshot testing-sum 1`] = `9`;

exports[`snapshot testing-sum 2`] = `3`;

exports[`snapshot testing-sum 3`] = `8`;

exports[`snapshot testing-sum 4`] = `9`;

exports[`snapshot testing-sum 5`] = `9`;
